flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/delegation/PlannerBase.scala

```diff
@@ -192,7 +192,11 @@ abstract class PlannerBase(
                 s"${classOf[OverwritableTableSink].getSimpleName} but actually got " +
                 sink.getClass.getName)
           }
-          LogicalSink.create(input, sink, catalogSink.getTablePath.mkString("."))
+          LogicalSink.create(
+            input,
+            sink,
+            catalogSink.getTablePath.mkString("."),
+            catalogManager.getTable(identifier).get().asInstanceOf[CatalogTable])
```

Comment:
try to reuse the `CatalogTable` you got above.

Reply:
You didn't resolve this.

Reply:
Can you check again? I have modified it.

Reply:
Sorry i didn't refresh the web